bot/code_review_bot/backend.py

```diff
@@ -131,8 +142,9 @@ def create(self, url_path, data):
         url_post = urllib.parse.urljoin(self.url, url_path)
         response = requests.post(url_post, json=data, auth=auth)
         if not response.ok:
-            logger.warn("Backend rejected the payload: {}".format(response.content))
-        response.raise_for_status()
+            logger.warning("Backend rejected the payload: {}".format(response.content))
+            # return an empty dict
+            return {}
```

Comment:
We should return `None` instead so it's clearer for callers that there was nothing created, and so that we skip this: https://github.com/mozilla/code-review/blob/9df3342394e283213f6598100e4607c1a934d2cf/bot/code_review_bot/__init__.py#L142

Reply:
Sure thing!